app/components/market/home/recent-template-list/recent-template-list.component.ts

```diff
@@ -25,14 +26,36 @@ export class RecentTemplateListComponent implements OnDestroy {
         this._subs = [];
     }
 
-    public formatMode(mode: Modes) {
+    public createParameterFile(recent: RecentSubmission) {
+        const dialog = remote.dialog;
+        const localPath = dialog.showSaveDialog({
+            buttonLabel: "Save",
+            defaultPath: this._getParameterFileName(recent.mode),
+        });
+
+        if (localPath) {
+            this.templateService.createParameterFileFromSubmission(localPath, recent);
+        }
+    }
+
+    public formatMode(mode: NcjTemplateMode) {
+        switch (mode) {
+            case NcjTemplateMode.NewPool:
+                return "Pool for later use";
+            case NcjTemplateMode.ExistingPoolAndJob:
+                return "Job with existing pool";
+            case NcjTemplateMode.NewPoolAndJob:
+                return "Job with auto-pool";
+        }
+    }
+
```

Comment:
I changed these to be more inline with the action that the user took when running the template. Hope you don't mind.